trailblazer/services/job_service.py

```diff
@@ -1,6 +1,6 @@
 from datetime import datetime
-from trailblazer.constants import TrailblazerStatus
 
+from trailblazer.constants import TrailblazerStatus
```

Comment:
@seallard I'm getting this diff due to imports not being sorted. Also, Ruff does not understand the init.py imports. Are you using the pre-commit tools?